tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Problem_Results.yaml

```diff
@@ -65,6 +66,31 @@ params:
   temporal_columns_lookup:
     emission_time: true
   time_grain_sqla: P1M
+  timeseries_limit_metric:
+    aggregate: AVG
+    column:
+      advanced_data_type: null
+      certification_details: null
+      certified_by: null
+      column_name: course_order
+      description: null
+      expression: null
+      filterable: true
+      groupby: true
+      id: 1348
+      is_certified: false
+      is_dttm: false
+      python_date_format: null
+      type: Int32
+      type_generic: 0
+      verbose_name: null
+      warning_markdown: null
+    datasourceWarning: false
+    expressionType: SIMPLE
+    hasCustomLabel: false
+    label: AVG(course_order)
```

Comment:
Any idea why were AVG this? I assume it needs to be in the group bym but something like a MAX might be quicker.